### PR TITLE
[DOCS] Retroactive Endpoint 8.1.1 release notes

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Fixes https://github.com/elastic/security-docs/issues/1760.

Preview [here](https://security-docs_1763.docs-preview.app.elstc.co/guide/en/security/master/release-notes-header-8.1.0.html#release-notes-8.1.1).

### CODE REVIEW

**Review by ferullo — APPROVED** [review]
Thanks!

File: docs/release-notes/8.1.asciidoc [hunk 1]
```diff
@@ -8,6 +8,7 @@
 [discrete]
 [[bug-fixes-8.1.1]]
 ==== Bug fixes and enhancements
+* Fixes an Endpoint Security integration bug that prevented benign Windows files from being deleted during certain circumstances.
```
Comment (joepeeples) [inline docs/release-notes/8.1.asciidoc:11]: Also, should there be a link to the bug fix PR?
```suggestion
* Fixes an {endpoint-sec} integration bug that prevented benign Windows files from being deleted during certain circumstances.
```

File: docs/release-notes/8.1.asciidoc [hunk 2]
```diff
@@ -8,6 +8,7 @@
 [discrete]
 [[bug-fixes-8.1.1]]
 ==== Bug fixes and enhancements
+* Fixes an {endpoint-sec} integration bug that prevented benign Windows files from being deleted during certain circumstances.
```
Comment (benironside) [inline docs/release-notes/8.1.asciidoc:11]:
```suggestion
* Fixes an {endpoint-sec} integration bug that prevented benign Windows files from being deleted under certain circumstances.
```

**Review by benironside — COMMENTED** [review]
Left one minor suggestion 👍

**Review by jmikell821 — APPROVED** [review]
LGTM! 🎉

**Pull request merged**